ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -218,11 +218,44 @@ public struct AvatarGroup: View, TokenizedControlView {
                    minHeight: groupHeight,
                    maxHeight: .infinity,
                    alignment: .leading)
+            .accessibilityElement(children: .combine)
+            .accessibilityLabel(groupLabel)
         }
 
         return avatarGroupContent
     }
 
+    var avatarsToDisplay: Int {
+        return min(state.maxDisplayedAvatars, state.avatars.count)
+    }
+
+    var displayedAvatarAccessibilityLabels: [String] {
+        var labels: [String] = []
+        for i in 0..<avatarsToDisplay {
+            let avatar = state.avatars[i]
+            labels.append(avatar.accessibilityLabel ?? avatar.primaryText ?? avatar.secondaryText ?? "")
+        }
+        let overflowCount = state.avatars.count - avatarsToDisplay + state.overflowCount
+        if overflowCount > 0 {
+            labels.append(String(format: "Accessibility.AvatarGroup.Overflow.Value".localized, overflowCount))
+        }
+        return labels
+    }
+
+    var groupLabel: String {
+        let displayedAvatarCount = displayedAvatarAccessibilityLabels.count
+        guard displayedAvatarCount > 1 else {
+            return displayedAvatarAccessibilityLabels.last ?? ""
+        }
+
+        var str: String = ""
+        for i in 0..<displayedAvatarAccessibilityLabels.count - 1 {
```

Comment:
Just use displayedAvatarCount instead of displayedAvatarAccessibilityLabels.count?

Reply:
Ah, thanks. Missed that one.